### ISSUE TITLE
Suggestion: Alternate way to display installed extensions in overflow menu

### ISSUE THREAD

**Issue by MmyGgithubAaccount** [kiwibrowser/src.next#74]
Can extensions be displayed in a compact manner like ungoogled chromium (screenshot attached)? This takes far less space and it's quicker to find and open extensions
![Screenshot_20210723-012703~2](https://user-images.githubusercontent.com/58236619/126710925-2b3b8406-3c68-4284-8c6b-d4618dec30c2.png)
Also if possible, a way to choose which extensions appear in the overflow menu and also reorder them would be nice

**Comment by Lartsch** [comment]
Good idea. Source from https://github.com/wchen342/chromium-android-extension can probably be reused.

**Comment by MmyGgithubAaccount** [comment]
@kiwibrowser thoughts on this and #72?

**Comment by kiwibrowser** [comment]
The license is GPL so it's not possible to use.
Though, a big part of this GPL code is actually taken from Kiwi itself, and they shouldn't be able to relicense it as GPL.

But in this situation it's better to write your own code.

**Comment by kiwibrowser** [comment]
The license is GPL so it's not possible to use it (see restricted licenses by Google: https://opensource.google/docs/thirdparty/licenses/#restricted )
Though, in practice ungoogled-chromium took a big part of the extensions code directly from Kiwi, so they shouldn't be able to relicense it as GPL.

In the current situation it's better to write your own code without looking at what exists already.

**Comment by kiwibrowser** [comment]
We can keep this issue on, it's not a problem to add this feature, just we should write the code ourselves :)

**Comment by kiwibrowser** [comment]
@KaKi87 because you seemed interested

**Comment by KaKi87** [comment]
I'm a web-only dev so I can't work on it however I agree that this feature request is still valid and I'm interested so let's keep it open :)
